xnat_admin_tools/remove_stale_data.py

```diff
@@ -35,7 +36,35 @@ def remove_stale(days: int):
 
     typer.echo(f"Found {len(to_delete)} sessions")
     for session in to_delete:
-        typer.echo(f"{session}")
+        typer.echo(f"Deleting {session}")
+        p = connection.select.project(session["project"])
+        p.subject(session["subject_id"]).delete()
+        assert not p.subject(
+            session["subject_id"]
+        ).exists(), "Participant data was not deleted"
```

Comment:
Asserts will raise an `AssertionError` if the expression is not true. Doing that here will break the loop. I don't think that's the intended effect

Reply:
I actually want it as full assertion, because this will be cronjob and if projects are not fully removed - something is not right with pyxnat and I should look into it